praw/models/reddit/redditor.py

```diff
@@ -204,6 +204,9 @@ def moderated(self):
         :returns: A ``list`` of :class:`~praw.models.Subreddit` objects.
             Return ``[]`` if the redditor has no moderated subreddits.
 
+        The redditor's own user profile subreddit will not be returned,
```

Comment:
Consider using `.. note` for this block.

Reply:
Done!